ds701_book/_quarto.yml

```diff
@@ -1,51 +1,54 @@
 project:
-  type: book
-  output-dir: _book
+  type: website
+  output-dir: _site
 
-book:
+website:
   title: "Tools for data science"
-  author: "Thomas Gardos and Scott Ladenheim"
-  date: "6/26/2024"
-  chapters:
-    - index.qmd
-    - part: "Preliminaires"
-      chapters:
+  back-to-top-navigation: true
+  sidebar:
+    style: "docked"
+    search: true
+    contents:
+    - href: index.qmd
+      text: Home
+    - section: "Preliminaires"
```

Comment:
This is a lingering typo from me that we should fix.

Reply:
fixed